### PR TITLE
Update the login and improve error messages

### PR DESCRIPTION
**Pull request by IvaKop**
Error messages are appropriate when credentials are wrong or there is no connection.
The authentication is updated with the mobile version with a token that does not expire.

### CODE REVIEW

File: src/redux/actions.js [hunk 1]
```diff
@@ -18,17 +23,20 @@ export const login = (username, password, env) => dispatch =>
...
-        Authorization: 'Basic YmFyQ2xpZW50SWRQYXNzd29yZDpzZWNyZXQ='
+        Authorization: 'Basic bW9iaWxlQ2xpZW50SWQ6bW9iaWxlQ2xpZW50U2VjcmV0'
       }
     }
   )
     .then(data => {
       if (data.status !== 200) {
-        throw new Error()
+        dispatch(setLoginError(null, data.status, null))
+        throw new Error(e)
```
Comment (Danman7) [inline src/redux/actions.js]: Where does this `e` come from? It will be `undefined`.

File: src/redux/actions.js [hunk 2]
```diff
@@ -18,17 +23,20 @@ export const login = (username, password, env) => dispatch =>
     {
       method: 'POST',
       headers: {
-        Authorization: 'Basic YmFyQ2xpZW50SWRQYXNzd29yZDpzZWNyZXQ='
+        Authorization: 'Basic bW9iaWxlQ2xpZW50SWQ6bW9iaWxlQ2xpZW50U2VjcmV0'
```
Comment (Danman7) [inline src/redux/actions.js]: The other actions use `headers: { Authorization: 'Bearer ${token}' }`.

Comment (IvaKop) [reply src/redux/actions.js]: This is the initial authentication and the value here is static.

Comment (Danman7) [reply src/redux/actions.js]: There should be no authorisation required for somеthing like login. But I understand.

**Pull request merged**